bedrock/firefox/templates/firefox/sync.html

```diff
@@ -212,9 +214,12 @@
   <div class="inner-wrapper">
     <div class="container">
       <header>
-        <h3>{{ _('Have an <b>Android smartphone</b> or <b>tablet</b>?') }}</h3>
-        <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" class="sync-button green" data-interaction="button click" data-element-action="Sync CTA" id="cta-android-footer">
-          {{ _('DOWNLOAD FIREFOX FOR ANDROID') }}
+        <h3>{{ _('Get Firefox everywhere') }}</h3>
```

Comment:
@flodolo - This string exists on `/firefox/new`, though it is behind an `l10n` tag. However, according to the .lang files checker(http://l10n.mozilla-community.org/~pascalc/langchecker/?website=0&file=firefox/new.lang&action=translate&show), that string is translated for 92% of our user base.

I'm guessing the answers below are both "yes", but just to be explicit:

1. Is that a high enough percentage to merge as-is?

2. Should I `add_lang_files "firefox/new"` for this page?